packages/app-check/src/providers.ts

```diff
@@ -88,6 +89,66 @@ export class ReCaptchaV3Provider implements AppCheckProvider {
   }
 }
 
+/**
+ * App Check provider that can obtain a reCAPTCHA Enterprise token and exchange it
+ * for an App Check token.
+ *
+ * @public
+ */
+export class ReCaptchaEnterpriseProvider implements AppCheckProvider {
+  private _app?: FirebaseApp;
+  private _platformLoggerProvider?: Provider<'platform-logger'>;
+  /**
+   * Create a ReCaptchaV3Provider instance.
+   * @param siteKey - ReCAPTCHA V3 siteKey.
```

Comment:
Should say "reCAPTCHA Enterprise score-based site key."

Reply:
And the line above that should say "Create a ReCaptchaEnterpriseProvider instance."

Reply:
Fixed.